crud/src/main/java/com/redhat/lightblue/mediator/Mediator.java

```diff
@@ -115,7 +116,13 @@ public Mediator(Metadata md,
      * implementation can perform further validations.
      */
     @StopWatch(loggerName = "stopwatch.com.redhat.lightblue.mediator.Mediator", sizeCalculatorClass = "com.redhat.lightblue.mediator.ResponsePayloadSizeCalculator")
-    public Response insert(InsertionRequest req) {
+    public Response insert(InsertionRequest req, RequestMetrics metrics) {
+        RequestMetrics.Context metricCtx;
+        if (metrics.isBulkRequest()) {
+            metricCtx = metrics.startBulkRequest("insert", req.getEntityVersion().getEntity(), req.getEntityVersion().getVersion());
```

Comment:
I think if the request is trying to use the default version, this will just have `null` for version. If we initialize this after lightblue has resolved the version, we can get the version that is actually used, which is more useful. I *think* you can get this from below like

```
EntityMetadata md = ctx.getTopLevelEntityMetadata();
md.getVersion().getValue(); // <-- this is the resolved version
```